crates/masonry/src/text2/edit.rs

```diff
@@ -52,27 +52,31 @@ impl EditableText for String {
 
 /// A region of text which can support editing operations
 pub struct TextEditor<T: EditableText> {
-    selection: TextWithSelection<T>,
+    with: TextWithSelection<T>,
```

Comment:
Daniel suggests "inner" as a better name for this, and I agree. I realize it's tricky to come up with a perfect name for this.